### ISSUE TITLE
Enhance: Item slots

### ISSUE THREAD

**Issue by Nyyrazzilyss** [torilmud/issues#819]
Item slots with multiple choice (ear, neck, wrist, weapon) were update to provide unique names. The helpfile for 'remove' was updated to show this, but not the matching helpfile for 'wear'.

It would also be nice if that could also be included in the response from the mud when you type equip. Example, returning:

worn in ear        an earring of the fires (magic)
worn in ear2      a dangling batskull earring

( github didn't like the brackets)

**Comment by shevarash** [comment]
Good idea. I don't love displaying it as [ear2] though. Maybe [right ear] and [left ear].

**Comment by Nyyrazzilyss** [comment]
Sure, right/left ear, right/left wrist would work. Only hard one would be neck: upper/lower neck?
